server/controllers/authentication.coffee

```diff
@@ -12,18 +12,33 @@ otpManager   = require '../lib/2fa_manager'
 
 
 getEnv = (callback) ->
-    User.getUsername (err, username) ->
-        return callback err if err
+    async.parallel(
+        env: (callback) ->
```

Comment:
@m4dz you should rename `callback` in `done`, to avoid confusion with the callback from `getEnv`